src/pages/invoices/create/hooks/useInvoiceUtilities.ts

```diff
@@ -69,8 +69,8 @@ export function useInvoiceUtilities(props: Props) {
 
     if (currency && invoice) {
       const invoiceSum = invoice.uses_inclusive_taxes
-        ? new InvoiceSumInclusive(invoice, currency).build()
-        : new InvoiceSum(invoice, currency).build();
+        ? new InvoiceSumInclusive({ ...invoice }, currency).build()
+        : new InvoiceSum({ ...invoice }, currency).build();
```

Comment:
I wonder if this is still required? Isn't the spread operator part of previous implementation?

Reply:
@beganovich Yes, you are right, this is unnecessary until we improve the implementation. So, I've removed the spreading. Along with this removal, I've fixed conflicts related to merging other PRs. Let me know your thoughts.